src/commands/basemaps-github/create-pr.ts

```diff
@@ -80,9 +80,20 @@ async function parseVectorTargetInfo(target: string): Promise<{ name: string; ti
 
   if (epsg == null || name == null) throw new Error(`Invalid target ${target} to parse the epsg and imagery name.`);
   if (epsg !== Epsg.Google) throw new Error(`Unsupported epsg code ${epsg.code} for vector map.`);
-  // Try to get the region for individual layers
+  // Try to get the title
+  const collectionPath = fsa.join(target, 'collection.json');
+  const collection = await fsa.readJson<StacCollection>(collectionPath);
+  if (collection == null) throw new Error(`Failed to get target collection json from ${collectionPath}.`);
+  const ldsLayers = collection.links.filter((f) => f.rel === 'lds:layer');
+  let title = collection.title;
+  // Get title from lds:title for individual vector layer
+  if (ldsLayers.length === 1) {
+    const ldsTitle = ldsLayers[0]?.['lds:title'];
```

Comment:
I don't think `?` here is necessary, because the previous line ensures that index 0 exists. Unless you could have a layer which is literally `undefined`, which would probably be a bug.

Reply:
I was trying to remove ? this earlier, but the type checker doesn't allows to. Seems the filter returns a `unknown` type from the filter(line 87), which can be `undifned` type as an `unknown`.

Reply:
I think the standard solution for that would be to declare the type of `ldsLayers`. But since that's more verbose, let's just go with this for now.